pkg/proc/eval.go

```diff
@@ -2064,6 +2067,33 @@ func (scope *EvalScope) evalIndex(op *evalop.Index, stack *evalStack) {
 		return
 	}
 
+	if xev.Name == "runtime."+evalop.BreakpointHitCountVarName {
```

Comment:
We do this construction of `runtime.bphitcount` repeatedly, perhaps we just have a constant for the fully qualified name.

Reply:
Done.